cpu/esp8266/include/user_config.h

```diff
@@ -13,10 +20,15 @@
 extern "C" {
 #endif
 
-/* This file is just to satisfy SDK */
+/**
+ * @brief   Default CPU frequency in MHz.
+ * Possible values are 80 and 160.
+ */
+#define ESP8266_CPU_FREQUENCY_DEFAULT   80
```

Comment:
Why do you define a default and then check if `ESP8266_CPU_FREQUENCY` is defined instead of defining it like..?
```C
#ifndef ESP8266_CPU_FREQUENCY
#define ESP8266_CPU_FREQUENCY   80
#endif
```
That way you wouldn't need the `#ifdef...` on the `user_init` function. Is there something I'm missing?